tests/filecheck/mlir-conversion/llvm_types.xdsl

```diff
@@ -0,0 +1,21 @@
+// RUN: xdsl-opt -t mlir %s | filecheck %s
+
+
+module() {
+  func.func() ["sym_name" = "struct_to_struct", "function_type" = !fun<[!llvm.struct<"", [!i32]>], [!llvm.struct<"", [!i32]>]>, "sym_visibility" = "private"] {
+    ^0(%0 : !llvm.struct<"", [!i32]>):
+      func.return(%0 : !llvm.struct<"", [!i32]>)
+  }
+  func.func() ["sym_name" = "struct_to_struct2", "function_type" = !fun<[!llvm.struct<"", [!i32, !i32]>], [!llvm.struct<"", [!i32, !i32]>]>, "sym_visibility" = "private"] {
+    ^1(%1 : !llvm.struct<"", [!i32, !i32]>):
+      func.return(%1 : !llvm.struct<"", [!i32, !i32]>)
+  }
+}
+
+
+//      CHECK: func.func private @struct_to_struct(%arg0: !llvm.struct<(i32)>) -> !llvm.struct<(i32)> {
```

Comment:
Can you change the explicit value names to `%{{.*}}` ? So if there is some name change in LLVM this won't break